.changeset/common-goats-exist.md

```diff
@@ -0,0 +1,25 @@
+---
+'@backstage/plugin-notifications-backend': patch
+---
+
+Notifications are now automatically deleted after 1 year by default.
+
+There is a new scheduled task that runs every 24 hours to delete notifications older than 1 year.
+This can be configured by setting the `notifications.expiration.period` in the `app-config.yaml` file.
+
+```yaml
+notifications:
+  expiration:
+    period: 365d # Default is 365 days
```

Comment:
you call it period in this file, but duration in config.d.ts.

Completely alternative suggestion:

```yaml
notifications:
  retention: 1y
```

?